functions/combine_data.R

```diff
@@ -72,32 +66,32 @@ combine_data <-
     discovery$WaitWeeks <- str_extract(discovery$WaitWeeks, "[0-9]+")
     
     
-    ### 4 - Write dataframes to an Excel file in 4 locations ----
+    ### 4 - Write dataframes to 2 Excel files ----
     
     # Set path for exporting to \\nssstats01.csa.scot.nhs.uk\WaitingTimes\Chronic-Pain\Data\Database
-    path_1 = "//PHI_conf/WaitingTimes/Chronic-Pain/Data/Database/"
+    #path_1 = "//PHI_conf/WaitingTimes/Chronic-Pain/Data/Database/"
```

Comment:
This is currently set up as a variable, `path_alldata` from the `setup_environment` script. Avoid static file paths as much as possible, and when they're necessary make sure they're not hidden within code. This is true for paths following this too.